.github/workflows/dispatch-ci.yml

```diff
@@ -0,0 +1,16 @@
+name: Dispatch CI
```

Comment:
This file was added because I changed the target branch from `4` to `4.13` - it's fine to backport this file, it'll have no impact